CHANGES.rst

```diff
@@ -24,7 +24,7 @@
 Changes
 =======
 
-Version 1.0.0a11 (released 2017-10-16)
+Version 1.0.0b1 (released 2017-10-23)
 --------------------------------------
 
 Incompatible changes
```

Comment:
Leaving incompatible changes for beta release is a bit weird, can we rephrase to something like [Initial public release](https://github.com/inveniosoftware/invenio-oauthclient/blob/master/CHANGES.rst#version-100b2-released-2017-08-10)? WDYT? :)

Reply:
Good point, I've updated it.